app.py

```diff
@@ -33,38 +33,38 @@ def __init__(self, cfg):
 
         self.test_transforms = transforms.Compose([transforms.Resize((224, 224), interpolation=2),
             transforms.ToTensor(),
-            transforms.Normalize(mean=[0.5, 0.5, 0.5],
-                std=[0.5, 0.5, 0.5])])
+            transforms.Normalize(mean=[0.5, 0.5, 0.5], std=[0.5, 0.5, 0.5])])
```

Comment:
is this right 0.5 value? @Magauiya @azamatkhid

Reply:
@flrngel those mean and std values were inspired by:

https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html

Reply:
@flrngel it depends on a dataset type. For toy datasets like Fashion MNIST, CIFAR should be fine, but ImageNet requires its own settings. In my opinion, **mean** and **variance** should be calculated from the dataset.